### DISCUSSION TITLE
Nimib Speaking Hours June 2023 - Saturday 10th

### DISCUSSION THREAD

**Discussion by pietroppeter** [pietroppeter/nimib#207] [Nimib speaking hours]
**update**: it's settled for Saturday morning (10 CEST), here is a link to add it to your calendar: https://calndr.link/event/FIR7M9Unkd (as usual we will connect on jitsi).

Hi @HugoGranstrom and others (@dlesnoff [expressed interest](https://github.com/pietroppeter/nimib/discussions/200#discussioncomment-5688012) but maybe also @neroist might be interested and anyone actually is invited as usual... :)), what do you say if we do a nimib speaking hours this weekend? I would propose Saturday at 10AM CEST but I am open to other options if someone prefers another time or day (Sunday instead of Saturday). Agenda will be to pick up where we left. As anticipated (actually a bit worse than anticipated) I had very little time these last two months to work on nimib but now the situation looks better.

**Comment by HugoGranstrom** [comment]
I will be able to join, but I will only be able to stay for 1h before I will have to leave 😄

**Comment by dlesnoff** [comment]
I will be able to join on Saturday morning!

Sincerely,
Dimitri

Le mar. 6 juin 2023, 18:30, Pietro Peterlongo ***@***.***> a
écrit :

**Comment by pietroppeter** [comment]
update the first post with a link to the calendar event, let me know if there are mistakes...

**Comment by pietroppeter** [comment]
Notes from the meeting:
- present me, @HugoGranstrom and we welcomed @dlesnoff !
- started by asking what dimitri is interested about:
  - future of nimib?
  - which issue can I contribute to?
- on the future of nimib we shared what we are currently working on:
  - pietro:
    - static site generator for nimib
      - first step is work going on in https://github.com/pietroppeter/nblog/pull/16
      - main idea is to add an intermediate step: nim -> json -> html
      - some of the work there will be ported to nimib (a --nbJson switch for outputing json and a way to recover a html from a json)
      - next the idea is to port https://github.com/dmulholl/ark (formerly ivy) to nim
        - name could be snark (which is the sound you make while sleeping in swedish)
  - hugo
    - improving docs for nimislides
    - nimibex
- on issues for dimitri:
  - https://github.com/pietroppeter/nimib/issues/194
  - https://github.com/pietroppeter/nimib/issues/101

we discussed a bit more on some of the points above, is someone in interest in one of those, just ask and we can add more details.

Have a great weekend everyone!

**Reply by dlesnoff** [reply]
Thanks for the meeting notes!

**Reply by pietroppeter** [reply]
Thank you for participating and contributing! 😀
